psa-attestation/Cargo.toml

```diff
@@ -18,8 +18,11 @@ nitro = []
 
 [dependencies]
 libc = "0.2.69"
+# We are not really using psa-crypto but we are using the C libraries
+# libmbedcrypto.a and libshim.a that psa-crypto indirectly uses.
+psa-crypto = { path = "../third-party/rust-psa-crypto/psa-crypto" }
 
 [build-dependencies]
-bindgen = "0.59"
+bindgen = "0.57.0"
```

Comment:
The `icecap-sel4-sys` crate uses `bindgen = "*"` (which is currently `0.59.2`) should we all align on 0.57 or 0.59?

Reply:
`0.57` is what I've tested!

Reply:
I've changed that to `0.59.2`.